ddocs/medic-client/views/messages_by_contact_date/map.js

```diff
@@ -1,71 +1,37 @@
 function(doc) {
-  var getPosition = function(facility) {
-    if (facility) {
-      var nameParts = [];
-      while (facility) {
-        if (facility.name) {
-          nameParts.push(facility.name);
-        }
-        facility = facility.parent;
-      }
-      if (nameParts.length) {
-        return nameParts;
+
+  var getLineage = function(contact) {
```

Comment:
Yeah absolutely. I've left it duplicated here for now but once both our merges are in it'd be great to pull it out into a shared lib.